src/Wallabag/CoreBundle/Controller/FeedController.php

```diff
@@ -137,11 +150,28 @@ public function showTagsFeedAction(User $user, Tag $tag, $page)
                 'domainName' => $this->getParameter('domain_name'),
                 'version' => $this->getParameter('wallabag_core.version'),
                 'tag' => $tag->getSlug(),
+                'updated' => $this->prepareFeedUpdatedDate($entries, $sort),
             ],
             new Response('', 200, ['Content-Type' => 'application/atom+xml'])
         );
     }
 
+    private function prepareFeedUpdatedDate(Pagerfanta $entries, $sort = 'created')
+    {
+        $updated = null;
+        $currentPageResults = $entries->getCurrentPageResults();
+        if (isset($currentPageResults[0])) {
+            $firstEntry = $currentPageResults[0];
+            if ($sort == 'created') {
+                $updated = $firstEntry->getCreatedAt();
+            } else {
+                $updated = $firstEntry->getUpdatedAt();
+            }
+        }
+
+        return $updated;
```

Comment:
Return early instead of using an `else`
```suggestion
        $currentPageResults = $entries->getCurrentPageResults();
        if (isset($currentPageResults[0])) {
            $firstEntry = $currentPageResults[0];
            if ($sort == 'created') {
                return $firstEntry->getCreatedAt();
            }

            return $firstEntry->getUpdatedAt();
        }

        return null;
```